### PR TITLE
Replace and/or remove refrences to ludeeus

### PR DESCRIPTION
**Pull request by ludeeus**
# Proposed Changes

> (Describe the changes and rationale behind them)

It's now been almost 3 years since my last commit to this repository, which means that I'm clearly not maintaining it, and the credit for that should not be given to me.

## Related Issues

> ([Github link][autolink-references] to related issues or pull requests)

[autolink-references]: https://help.github.com/articles/autolinked-references-and-urls/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,7 +70,7 @@ our [GitHub Repository][repository].
 
 MIT License
 
-Copyright (c) 2018-2022 Joakim Sørensen
+Copyright (c) 2018-2022 Home Assistant Community Add-ons project
```
Comment (frenck) [inline README.md:73]:
```suggestion
Copyright (c) 2018-2019 Joakim Sørensen
Copyright (c) 2019-2002 Franck Nijhof
```

File: sqlite-web/Dockerfile [hunk 2]
```diff
@@ -60,11 +60,11 @@ LABEL \
     io.hass.arch="${BUILD_ARCH}" \
     io.hass.type="addon" \
     io.hass.version=${BUILD_VERSION} \
-    maintainer="Joakim Sørensen @ludeeus <ludeeus@gmail.com>" \
+    maintainer="Home Assistant Community Add-ons project" \
```
Comment (frenck) [inline sqlite-web/Dockerfile:63]:
```suggestion
    maintainer="Franck Nijhof <frenck@addons.community>" \
```

File: sqlite-web/Dockerfile [hunk 3]
```diff
@@ -60,11 +60,11 @@ LABEL \
...
     org.opencontainers.image.title="${BUILD_NAME}" \
     org.opencontainers.image.description="${BUILD_DESCRIPTION}" \
     org.opencontainers.image.vendor="Home Assistant Community Add-ons" \
-    org.opencontainers.image.authors="Joakim Sørensen @ludeeus <ludeeus@gmail.com>" \
+    org.opencontainers.image.authors="Home Assistant Community Add-ons project" \
```
Comment (frenck) [inline sqlite-web/Dockerfile:67]:
```suggestion
    org.opencontainers.image.authors="Franck Nijhof <frenck@addons.community>" \
```

**Review by frenck — COMMENTED** [review]
Is this ok with you?

File: README.md [hunk 4]
```diff
@@ -72,7 +70,8 @@ our [GitHub Repository][repository].
 
 MIT License
 
-Copyright (c) 2018-2022 Joakim Sørensen
+Copyright (c) 2018-2019 Joakim Sørensen
+Copyright (c) 2019-2002 Franck Nijhof
```
Comment (ludeeus) [inline README.md:74]:
```suggestion
Copyright (c) 2019-2022 Franck Nijhof
```

**Review by frenck — APPROVED** [review]
Thanks @ludeeus 👍

**Pull request merged**